README.md

```diff
@@ -1,38 +1,16 @@
 # obal - packaging wrapper using ansible
 
-## Updating a Package from Upstream
+`obal` is an ansible wrapper with a set of ansible playbooks to ease maintanance of packaging repositories like [`foreman-packaging`](https://github.com/theforeman/foreman-packaging) and [`pulp-packaging`](https://github.com/pulp/pulp-packaging).
 
-To update a package from upstream, start by updating the `package_manifest.yml` entry for that package with the new versions and relevant information. Next run the update action specifying the package:
-
-    obal update tfm-rubygem-hammer_cli
-
-This is equivalent to running the `update_package.yml` playbook:
-
-    ansible-playbook --inventory package_manifest.yaml /path/to/obal/update_package.yml -l tfm-rubygem-hammer_cli
-
-You can review the changes made by doing a `git status`. After making any manual changes, commit the changes and open an MR with the updates.
-
-## Updating a Package from Downstream
+All `obal` actions should also work with plain ansible when called like `ansible-playbook <action_plaabook>.yml -l <package>` instead of `obal <action> <package>`.
 
-To update a package from downstream, no changes to `package_manifest.yml` are needed, just run the update action specifying the package and the correct downstream version:
-
-    obal update tfm-rubygem-hammer_cli -e downstream_version=0.11.0.1
-
-This is equivalent to running the `update_package.yml` playbook:
+## available action (playbooks)
 
-    ansible-playbook --inventory package_manifest.yaml /path/to/obal/update_package.yml -l tfm-rubygem-hammer_cli -e downstream_version=0.11.0.1
+### add (add_package.yml)
```

Comment:
Mind seeing what one header level down would look like? Visually each one is a bit too big for me with 3